src/tmp.jl

```diff
@@ -418,28 +427,34 @@ function tmp_create_highest_resolution_table(connection)
     # - keep all unique time_block_start
     # - create corresponing time_block_end
 
+    # filtering by active
     for union_table in
         ("t_union_" * x for x in ("in_flows", "out_flows", "assets_and_out_flows", "all_flows"))
         table_name = replace(union_table, "t_union" => "t_highest")
         DuckDB.execute(
             connection,
             "CREATE OR REPLACE TABLE $table_name AS
             SELECT
-                asset,
+                t_union.asset,
                 t_union.year,
                 t_union.rep_period,
-                time_block_start,
-                lead(time_block_start - 1, 1, rep_periods_data.num_timesteps)
-                    OVER (PARTITION BY asset, t_union.year, t_union.rep_period ORDER BY time_block_start)
+                t_union.time_block_start,
+                lead(t_union.time_block_start - 1, 1, rep_periods_data.num_timesteps)
+                    OVER (PARTITION BY t_union.asset, t_union.year, t_union.rep_period ORDER BY t_union.time_block_start)
                     AS time_block_end
             FROM (
                 SELECT DISTINCT asset, year, rep_period, time_block_start
                 FROM $union_table
             ) AS t_union
+            LEFT JOIN asset_both
+                ON asset_both.asset = t_union.asset
+                AND asset_both.milestone_year = t_union.year
+                AND asset_both.commission_year = t_union.year
             LEFT JOIN rep_periods_data
                 ON t_union.year = rep_periods_data.year
                     AND t_union.rep_period = rep_periods_data.rep_period
-            ORDER BY asset, t_union.year, t_union.rep_period, time_block_start
+            WHERE asset_both.active = true
```

Comment:
Just for my understanding, the JOIN here is to have available in the table the `active` parameter. So, when we take out `active` from the Tulipa files in #960 we don't have to do this JOIN. Is that correct?

Reply:
The stored table itself won't have the `active` parameter because I haven't selected it, but the intermediary table resulting from the join will, which is then filtered in this `WHERE`. So the `JOIN` is only so that we have `active` available for filtering.

Reply:
So, deleting the active also will also delete this extra join and make the code more simple 😄 

Thanks!

Reply:
During #960, this will also be removed as there will be no mention of `active` anywhere. I was thinking about where the removal of `active` will happen, and the earliest point would be not having `active` at all in the input of TulipaEnergyModel. That means that during the workflow processing, at some point we check for `active` and remove. In the current workflow, it would be at the `read_csv_folder` function in TulipaIO. It's not clear yet the implications, and it's not urgent, so I'm not actively thinking about it.
For this PR, I noticed the constraint creations had a bunch of active checks, and the constraints could look a bit simpler if they didn't, so I shifted left.